### DISCUSSION TITLE
nrf5340 application example

### DISCUSSION THREAD

**Discussion by ghost** [hathach/tinyusb#2008] [General]
Hi,
Many files exist in the TinyUSB package to support the nrf53. However, there is no support for a real board (i.e. the DK pca10095).
I tried to build my custom implementation, but I am facing to the incompatibility between the existing nrf52 to the nrf53 (i.e. the power seems managed between the 2 CPU in a different way). So, no way, the package to fully support the nrf53 is not completed.
Has someone an example of board adaptation (i.e. the Nordic DK pca10095) that run TinyUSB?
Thank you,
Edo

**Comment by hathach** [comment]
The main reason is I don't have pca10095 to add the bsp. Actually I have an pca10095 pdk which does not have usb connector populated (waste of money on my side). Anyway, I think you should check out mynewt code base, I think they have out of the box suppport for nrf53, you could borrow some of their setup/power code there.

**Comment by ghost** [comment]
Hi,
If the main reason is that you do not have a pca10095 with the usb connector, 
Just give me your postal address that I will send to you a fully mounted pca10095.

Regards,
Edo

**Reply by hathach** [reply]
@EdoFranzi please drop me an email to thach@tinyusb.org for further discussion.
